### PR TITLE
Made small adjustments to the readme file

### PR DESCRIPTION
**Pull request by zambroid**
The readme file was containing small typos, I tried to identify them and my proposed new version of the file is here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ T-Pot offers docker images for the following honeypots ...
 * [Cockpit](https://cockpit-project.org/running) for a lightweight and secure WebManagement and WebTerminal.
 * [Cyberchef](https://gchq.github.io/CyberChef/) a web app for encryption, encoding, compression and data analysis.
 * [Elastic Stack](https://www.elastic.co/videos) to beautifully visualize all the events captured by T-Pot.
-* [Elasticvue](https://github.com/cars10/elasticvue/) a web front end for browsing and interacting with an Elastic Search cluster.
+* [Elasticvue](https://github.com/cars10/elasticvue/) a web front end for browsing and interacting with an ElasticSearch cluster.
```
Comment (t3chn0m4g3) [inline README.md:130]: Officially it is `Elasticsearch`

File: README.md [hunk 2]
```diff
@@ -156,9 +156,9 @@ T-Pot offers a number of services which are basically divided into five groups:
     * Logstash for ingesting, receiving and sending events to Elasticsearch.
     * Kibana for displaying events on beautifully rendered dashboards.
 3. Tools
-    * NGINX for providing secure remote access (reverse proxy) to Kibana, CyberChef, Elasticvue, GeoIP AttackMap and Spiderfoot.
+    * NGINX provides secure remote access (reverse proxy) to Kibana, CyberChef, Elasticvue, GeoIP AttackMap and Spiderfoot.
     * CyberChef a web app for encryption, encoding, compression and data analysis.
-    * Elasticvue a web front end for browsing and interacting with an Elastic Search cluster.
+    * Elasticvue a web front end for browsing and interacting with an ElasticSearch cluster.
```
Comment (t3chn0m4g3) [inline README.md:161]: Elasticsearch

File: README.md [hunk 3]
```diff
@@ -269,20 +269,20 @@ Besides the ports generally needed by the OS, i.e. obtaining a DHCP lease, DNS,
...
+Once you are familiar with how things work you should choose a network you suspect intruders in or from (i.e. the internet). Otherwise T-Pot will most likely not capture any attacks (unless you want to prove a point)! For starters it is recommended to put T-Pot in an unfiltered zone, where all TCP and UDP traffic is forwarded to T-Pot's network interface. To avoid probing for T-Pot's management ports you can put T-Pot behind a firewall and forward all TCP / UDP traffic in the port range of 1-64000 to T-Pot while allowing access to ports > 64000 only from trusted IPs and / or only expose the [ports](#required-ports) relevant to your use-case. If you wish to catch malware traffic on unknown ports you should not limit the ports you forward since glutton and honeytrap dynamically bind any TCP port that is not covered by other honeypot daemons and thus give you a better representation of what risks your setup is exposed to.
```
Comment (t3chn0m4g3) [inline README.md:281]: ... of the risks your setup ...

File: README.md [hunk 4]
```diff
@@ -692,13 +692,13 @@ If you want to remove users you just modify `nginxpasswd` with `vi` or any other
 <br><br>
 
 ## Import and Export Kibana Objects
-Some T-Pot updates will require you to update the Kibana objects. Either to support new honeypots or to improve existing dashboards or visualizations. Make sure to ***export*** first so you do not loose any of your adjustments.
+Some T-Pot updates will require you to update the Kibana objects. Either to support new honeypots or to improve existing dashboards or visualizations. Make sure to ***export*** first so you do not lose any of your adjustments.
```
Comment (t3chn0m4g3) [inline README.md:695]: loose is correct

**Comment by t3chn0m4g3** [comment]
Thank you, can you please adjust the changes, then I will merge.

**Review by zambroid — COMMENTED** [review]
LGTM

**Pull request closed without merging**